README.md

```diff
@@ -1,2 +1,22 @@
-# atlponda.github.io
-ATL Ponda website.
+# ATL Ponda Website
+https://atlponda.github.io
+
+****
+
+### Authors
+
+- Harsh Khandeparkar - [**@HarshKhandeparkar**](https://github.com/HarshKhandeparkar)
+
```

Comment:
Add my name in author :cry:

Reply:
You can add it.

Reply:
You can commit to this branch

Reply:
I will do that later.

Reply:
So shall I mwrge this now? You can do that in a followup. It will also add a commit to your contributions.